lms/djangoapps/courseware/grades.py

```diff
@@ -735,16 +621,10 @@ def get_score(user, block, scores_client, submissions_scores_cache, max_scores_c
     # older version of the problem -- they're still graded on what was possible
     # when they tried the problem, not what it's worth now.
     score = scores_client.get(block.location)
-    cached_max_score = max_scores_cache.get(block.location)
     if score and score.total is not None:
         # We have a valid score, just use it.
         correct = score.correct if score.correct is not None else 0.0
         total = score.total
-    elif cached_max_score is not None and settings.FEATURES.get("ENABLE_MAX_SCORE_CACHE"):
```

Comment:
Remove all other occurrences of `ENABLE_MAX_SCORE_CACHE` in the codebase.  I see 3 others.